src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-application-migration-services-integration.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: AWS Application Migration Service integration
+tags:
+  - Integrations
+  - Amazon integrations
+  - AWS integrations list
+  - Application Migration
+metaDescription: "New Relic's Amazon Application Migration Service integration"
+freshnessValidatedDate: 2024-09-03
+---
+
+You can now use the AWS [Application Migration Service](https://docs.aws.amazon.com/mgn/latest/ug/what-is-application-migration-service.html) to install the New Relic agent on applications migrating into AWS.
+
+## Setup [#setup]
+
+To have AWS install the [New Relic Infrastructure agent](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux/), Create a Post-launch template through the AWS console following these steps:
+
+1. Open Application Migration Services from your [AWS console](https://console.aws.amazon.com/mgn/home)
+2. Open the desired Source server from the available list
+3. Select the `Post-launch settings` tab
+4. Select `Edit`, and add the `Install New Relic agent` action, and add in the expected variables 
+
+## Monitoring [#monitor]
+
+Once the setup is complete, your AWS application will be monitored through the New Relic infrastructure agent. 
+You can find the data available in the New Relic applicaiton by opening `All Entites` and locating the instance under the `Hosts` entities.
```

Comment:
```suggestion
To find your data in New Relic:
1. Go to <DNT>**[one.newrelic.com](https://one.newrelic.com/) > All entities**</DNT>. 
2. In the **Hosts** entity and locate your  AWS Application Migration Service instance.
```

@jcountsNR how are these instances named in the UI? Do they follow a certain syntax? I want to mention on step 2 so they know what to look for (yay if it's just named "AWS Application Migration Service").

Reply:
They are just in the hosts section, and they are named based on the application. There is no specific 'application migration instance'.

Basically this is a method for installing New Relic Agent onto an instance that is being migrated to AWS from another location. The monitoring itself is no different than the infra agent running on EC2.